src/components/shared/LanguageSelect.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+
+import { useCookies } from 'react-cookie';
+import { RiGlobalLine } from 'react-icons/ri';
+import Select, { components } from 'react-select';
+
+const customStyles = {
+  control: (provided: any) => ({
+    ...provided,
+    border: 'none',
+    boxShadow: '0px 0px 0px 1px rgba(146, 146, 146, 0.16)',
+    minHeight: '50%',
+    width: '90px',
+    fontSize: '14px',
+    overflow: 'hidden',
+    outline: 'none',
+    paddingLeft: '0px',
+    flexDirection: 'row-reverse',
+    '&:hover': {
+      boxShadow: '0px 0px 0px 1px #017E84',
+      color: '#017E84',
+    },
+  }),
+  menuList: (base: any) => ({
+    ...base,
+    padding: 0,
+  }),
+  option: (base: any, state: any) => ({
+    ...base,
+    backgroundColor: state.isSelected ? '#E0EEEF' : null,
+    fontSize: '14px',
+    color: '#333232',
+    '&:hover': {
+      backgroundColor: '#E0EEEF',
+    },
+  }),
+  singleValue: (baseSingke: any) => ({
+    ...baseSingke,
+    color: 'inherit',
+    padding: '0px',
+  }),
+  dropdownIndicator: (base: any) => ({
+    ...base,
+    display: 'flex',
+    alignContent: 'center',
+    fontSize: '22px',
+    color: 'inherit',
+    paddingLeft: '11px',
+    paddingRight: '6px',
+    '&:hover': {
+      color: '#017E84',
+    },
+  }),
+  valueContainer: (base: any) => ({
+    ...base,
+    padding: '0px',
+  }),
+};
+
+function DropdownIndicator(props: any) {
+  return (
+    components.DropdownIndicator && (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+      <components.DropdownIndicator {...props}>
+        <RiGlobalLine />
+      </components.DropdownIndicator>
+    )
+  );
+}
+
+const options = [
+  { value: 'en', label: 'English', shortLabel: 'ENG' },
```

Comment:
What about values of this langages when for example we have 'French selected'? Should those labels be translated or should we keep 'english' version?

Reply:
There was no info about labels should be here translated :)